packages/services/api/src/modules/audit-logs/resolvers/ProjectCreatedAuditLog.ts

```diff
@@ -0,0 +1,39 @@
+import { ProjectCreatedAuditLogResolvers } from '../../../__generated__/types';
+
+export const ProjectCreatedAuditLog: ProjectCreatedAuditLogResolvers = {
+  __isTypeOf: e => e.event_action === 'PROJECT_CREATED',
```

Comment:
Since we are using `__isTypeOf` approach, we might have to add `AuditLog` to this array [here](https://github.com/kamilkisiela/graphql-hive/blob/d030e2ef49fe0bdf01d47eb0e61046c1c0adedb7/codegen.mts#L44-L48).

The reason being by default, server preset uses the `__typename` approach for determining abstract type members, so there's a bit of manual work to use `__isTypeOf` or `__resolveType`. This is something on my mind to improve in the future